src/controllers/connectionManager.ts

```diff
@@ -497,6 +505,21 @@ export default class ConnectionManager {
 				connection.credentials.server,
 				result.errorMessage ? result.errorMessage : result.messages)
 		);
+		result.errorNumber
+		sendErrorEvent(
+			TelemetryViews.ConnectionPrompt,
+			TelemetryActions.CreateConnectionResult,
+			new Error (result.errorMessage),
+			false,
+			result.errorNumber.toString(),
```

Comment:
```suggestion
			result.errorNumber?.toString(),
```
Just in case there is no error number..